app/src/main/java/org/mozilla/rocket/nightmode/AdjustBrightnessDialog.kt

```diff
@@ -30,29 +29,6 @@ class AdjustBrightnessDialog : BaseActivity() {
         }
     }
 
-    object Intents {
```

Comment:
Please put them in nightmode module. create a kt file, put this object there.
IntentUtils is exploding now.